### ISSUE TITLE
comScore sign up form not working

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#4017]
**Describe the issue you encountered:** The sign up form at comScore does not work unless shields are down.

**Expected behavior:** it should work.

- Brave Version: 0.12.1 RC1

- Steps to reproduce:
    1. Open https://www.comscore.com/Insights/Presentations-and-Whitepapers/2015/The-2015-US-Mobile-App-Report
    2. Click "Download Whitepaper" blue button on the page
    3. Fill the form and click the submit button

- Screenshot if needed:

- Any related issues:

cc: @BrendanEich

**Comment by BrendanEich** [comment]
@bridiver here's another case that might be helped by your 3rd party cookie block change. Can you tell for sure if so? Thanks.

**Comment by privatzee** [comment]
@BrendanEich Nope, cookies are not a factor there. The form wants to POST to Eloqua (owned by Oracle, "Eloqua enables marketers to plan and execute personalized marketing automation campaigns.") The form works with 3rd party cookies blocked, but it needs _Allow Ads and Tracking_.

So they want to trade that whitepaper in return for contact info.

![image](https://cloud.githubusercontent.com/assets/22591438/19534453/d0e70490-9611-11e6-9418-6ccd51538bc0.png)
